nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -867,6 +867,18 @@ php.override {
       <link xlink:href="https://github.com/gollum/gollum/wiki/5.0-release-notes#migrating-your-wiki">here</link>.
      </para>
    </listitem>
+   <listitem>
+     <para>
+       <literal>rubyMinimal</literal> was removed due to being unused and
+       unusable. The default ruby interpreter includes JIT support, which makes
+       it reference it's compiler. Since JIT support is probably needed by some
+       Gems, it was decided to enable this feature with all cc references by
+       default, and allow to build a Ruby derivation without references to cc,
+       by setting <literal>JitSupport = false;</literal> in an overlay. See
```

Comment:
```suggestion
       by setting <literal>jitSupport = false;</literal> in an overlay. See
```

Reply:
Right. I also rebased once more to put the release notes into rl-2103.xml.